librz/core/cmd_descs/cmd_analysis.yaml

```diff
@@ -383,6 +383,104 @@ commands:
                     type: RZ_CMD_ARG_TYPE_RZNUM
                   - name: addr
                     type: RZ_CMD_ARG_TYPE_RZNUM
+      - name: afl
+        summary: List functions
+        subcommands:
+          - name: afl
+            summary: List all functions
+            cname: analysis_fcn_list
+            type: RZ_CMD_DESC_TYPE_ARGV_STATE
+            modes:
+              - RZ_OUTPUT_MODE_STANDARD
+              - RZ_OUTPUT_MODE_JSON
+              - RZ_OUTPUT_MODE_QUIET
+              - RZ_OUTPUT_MODE_RIZIN
+              - RZ_OUTPUT_MODE_TABLE
+            args: []
+          - name: afll
+            summary: List all functions (verbose)
+            cname: analysis_fcn_list_verbose
+            type: RZ_CMD_DESC_TYPE_ARGV_STATE
+            modes:
+              - RZ_OUTPUT_MODE_TABLE
+            args: []
```

Comment:
can we put this as RZ_OUTPUT_MODE_LONG of `afl`? Maybe there is some tricks to do to use the table from the RzCmdStateOutput, but it should be possible, I think.

Reply:
Or add `RZ_OUTPUT_MODE_TABLE_LONG`

Reply:
Ok, both std and long modes are table now.